var/spack/repos/spack_repo/builtin/packages/spatter/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Spatter(CMakePackage, CudaPackage):
+    """A microbenchmark for timing Gather/Scatter kernels on CPUs and GPUs."""
+
+    homepage = "https://github.com/hpcgarage/spatter"
+    git = "https://github.com/hpcgarage/spatter.git"
+
+    maintainers("plavin", "jyoung3131")
+
+    license("MIT", checked_by="plavin")
+
+    variant(
+        "backend",
+        default="openmp",
+        values=["serial", "openmp", "cuda"],
+        description="Configuration string",
+    )
+    variant("mpi", default=False, description="Enable MPI support")
+    variant("cuda_arch", default="none", multi=True, description="CUDA architecture")
+
+    conflicts(
+        "backend=cuda",
+        when="cuda_arch=none",
+        msg="CUDA architecture must be specified when CUDA support is enabled.",
+    )
+
+    version("develop", branch="spatter-devel")
+    version("main", branch="main", preferred=True)
+    version("2.1", tag="v2.1", commit="ec8923711f8dc21eedff7189f12b02eb06845d2f")
```

Comment:
Please move the `version` directives to after the `license` to conform to the "standard" order.

Reply:
Done